conda_forge_tick/auto_tick.py

```diff
@@ -659,6 +660,7 @@ def add_rebuild_migration_yaml(
             Cos7Config(),
             CrossCompilationForARMAndPower(),
             MPIPinRunAsBuildCleanup(),
+            QtQtMainMigrator(),
```

Comment:
This should be added only when `name` is equal to whatever the name you used in the conda-forge-pinning-feedstock PR so that only that PR will get the qt->qt_main transition.

Reply:
I'll have to revisit this tomorrow. its too late for me to read through a new codebase.